cmd/apiserver/main.go

```diff
@@ -52,12 +49,10 @@ func main() {
 		fmt.Fprintln(os.Stderr, pflag.CommandLine.FlagUsages())
 	}
 	// parse the CLI flags
-	if f := goflag.Lookup("logtostderr"); f != nil {
-		f.Value.Set("true") // log to stderr by default
-	}
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine) //slurp in glog flags
 	pflag.Parse()
 	//goflag.CommandLine.Parse([]string{}) //https://github.com/kubernetes/kubernetes/issues/17162
+	logger := logutil.NewLogger(pflag.CommandLine)
```

Comment:
The `Caller` func below can be deleted with this refactoring.

Reply:
Good catch